awscli/botocore/data/dsql/2018-05-10/completions-1.json

```diff
@@ -0,0 +1,92 @@
+{
+  "version": "1.0",
+  "resources": {
+    "Cluster": {
+      "operation": "ListClusters",
+      "inputParameters": null,
```

Comment:
`inputParameters` is modeled in the completer generator, though it's not currently used/generated, which is why value in `null`. Going to manually remove it for now and fix this in code.